Dockerfile.vault-worker

```diff
@@ -1,3 +1,23 @@
+#  ----------------------------------------------------------------------------------
+#  Copyright 2018 ForgeRock AS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#  @author: Alain Pulluelo, ForgeRock (created: June 14, 2018)
+#  @version: 0.2.2
```

Comment:
you may want to go back through and change all these versions to 1.0.0 again in the license

Reply:
OK done